app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
+
+  def return_recent_posts
+    Post.limit(3).order(created_at: :desc)
+  end

```

Comment:
- [ optional ] You really did a great job but ill recommend using the `order` method before adding the limit because the idea is to order data in descending order before taking the first 3 rows. So ill suggest changing it to something like
 ```
def return_recent_posts
    Post.order(created_at: :desc).limit(3)
end
```